README.md

```diff
@@ -91,10 +93,11 @@ Now, that you have configured the Salesforce streaming Inbound Endpoint, use the
 ```
 * connection.salesforce.replay: replay **enable** or **disable**. Enabling this will read the event ID stored in the Registry DB or from the text file stored in the local machine.
 * connection.salesforce.EventIDStoredFilePath:
-    - when replay is enabled, do not define any value for this property (i.e., keep it blank), to replay from the last event ID stored in the Registry DB (property-“eventID” resource path:connector/salesforce/event).
+    - when replay is enabled, do not define any value for this property (i.e., keep it blank), to replay from the last event ID stored in the config Registry DB (property- name of the salesforce object(Follow bellow example) resource path:connector/salesforce/event).
     - when replay is enabled, specify the directory path of a text file to start replaying from the event ID stored in it. 
     ```
-    In case of a failure, the default value will be used to retrieve IDs from current events. Create the “eventID” property in the Registry DB in the connector/salesforce/event resource path . 
+    Ex: Salesforce object is /topic/reading__e then property name is "reading__e".
```

Comment:
```suggestion
    E.g., if the Salesforce object is `/topic/reading__e`, the property name is `reading__e`.
```

E.g. is a short form for a latin term called exempli gratia. This term translates to "for example". We cannot use different abbreviations like "Ex." for this.

Reply:
fixed in #11